source/distributed/tria.cc

```diff
@@ -3228,6 +3245,39 @@ namespace parallel
     }
 
 
+    template <int dim, int spacedim>
+    void
+    exchange_refinement_flags(Triangulation<dim, spacedim> &tria)
```

Comment:
I don't see the declaration of this function.

Reply:
@quangx Can you move this into the anonymous namespace higher up in the file, maybe here: https://github.com/dealii/dealii/blob/29f9da0a3412c3530b2f1b93c0f59ab6d909546f/source/distributed/tria.cc#L568 ?